dGame/dComponents/PetComponent.cpp

```diff
@@ -258,8 +316,7 @@ void PetComponent::OnUse(Entity* originator) {
 	GameMessages::SendNotifyPetTamingPuzzleSelected(originator->GetObjectID(), bricks, originator->GetSystemAddress());
 
 	m_Tamer = originator->GetObjectID();
-	SetStatus(5);
-	Game::entityManager->SerializeEntity(m_Parent);
```

Comment:
Is this being serialized elsewhere in here to let clients know of the change?

Reply:
let me do some testing to verify